### DISCUSSION TITLE
Github Education Glitching

### DISCUSSION THREAD

**Discussion by sahal-mulki** [community/community#147797] [GitHub Education]
### Hi everyone,

I reapplied for Github Student Benefits previously, about a month back, and it got ACCEPTED after my old application expired. I originally applied a few years back, which was also ACCEPTED. But now, just a month after I applied and got accepted, my Copilot is on the free plan, instead of pro, and my recent application shows up as expired, even though it was just around a month old. However my Github account is still on Pro, instead of Free. What's going on?

![image](https://github.com/user-attachments/assets/9995eab7-312d-46d8-b52b-d93a9b5950d5)
![image](https://github.com/user-attachments/assets/ea8421e8-bcc3-4859-81d4-e4e9ddf170d5)
![image](https://github.com/user-attachments/assets/1a9857d9-d69a-4173-8441-996092d493c6)

**Comment by mooosakhan** [comment]
It's a good opportunity for the students who need GitHub copilot

**Reply by sahal-mulki** [reply]
?????

**Comment by Ravindulakmina** [comment]
It may be.I also applied again. They accepted it and made my account pro. But I saw somewhere. When applying, there is limited access to some things in the student pack.
If you can, go to github support.
